pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -1327,4 +1327,6 @@ self: super: {
   haskell-lsp_0_18_0_0 = super.haskell-lsp_0_18_0_0.override { haskell-lsp-types = self.haskell-lsp-types_0_18_0_0; };
   lsp-test_0_8_2_0 = (dontCheck super.lsp-test_0_8_2_0).override { haskell-lsp = self.haskell-lsp_0_18_0_0; };
 
+  glirc = doJailbreak super.glirc;
```

Comment:
Could you leave a comment here why `glirc` requires `doJailbreak`, and when you expect it to no longer require it?

This helps the nixpkgs maintainers know when overrides can be removed.

Reply:
I have added a comment; is it sufficient?